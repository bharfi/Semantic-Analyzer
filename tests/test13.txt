Main{
    S2
    {
        public: a, b;
        a = b;
    }
    a = b;
}