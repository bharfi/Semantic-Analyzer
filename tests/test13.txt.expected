S2.a = S2.b
?.a = ?.b